src/main/java/racing/domain/Car.java

```diff
@@ -0,0 +1,48 @@
+package racing.domain;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Car {
+
+  private final String name;
+  private int position = 0;
+  private final int START_RANDOM_NUMBER = 0;
+  private final int FINISH_RANDOM_NUMBER = 9;
+  private final int CHECK_NUMBER = 4;
+  private final String MOVE_FORWARD_CHECKPOINT = "-";
+
+
+  public Car(String name) {
+    this.name = name;
+  }
+
+  // 추가 기능 구현
+
+  public static Car createCar(String name) {
+    return new Car(name);
+  }
```

Comment:
Active Record Pattern인데 음 .. 장점과 단점 찾아보면 좋을듯!
```suggestion
  public static Car createCar(String name) {
    return new Car(name);
  }
```